### PR TITLE
Improve icon key.

### PR DESCRIPTION
**Pull request by BigBlueHat**
- Output all available icons in the key.
- Update CHANGELOG.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,14 @@
 # @digitalbazaar/mocha-w3c-interop-reporter ChangeLog
 
+## 1.x.x - 2024-mm-dd
+
+### Added
+- Tests can now provide `this.test.link` to link test names to a destination
```
Comment (aljones15) [inline CHANGELOG.md:6]: where is the code for this feature or was updating the changelog missed?

File: templates/test-statistics.hbs [hunk 2]
```diff
@@ -13,17 +13,16 @@
...
-✓ - Success
-      </li>
-      <li>
-❌ - Error
-      </li>
-      <li>
-— - Implementation explicitly does not support this feature
-      </li>
-    </ul>
+    <table>
```
Comment (aljones15) [inline templates/test-statistics.hbs:16]: programmatically showing all the statuses seems cool and a good idea approved.

**Review by aljones15 — APPROVED** [review]
I like the idea of the status lists being expandable by a json object so good idea here.

File: CHANGELOG.md (hunk 1 above)
Comment (BigBlueHat) [reply CHANGELOG.md:6]: @aljones15 I missed updating the changelog in #37.

**Comment by BigBlueHat** [comment]
@decentralgabe the icon key has all the things now (and will "formally" in the next release--depending on how you're including it), just FYI.

**Pull request merged**